src/Numerics.jl

```diff
@@ -201,6 +201,12 @@ function LinearAlgebra.svd(tensor::Tensor; left_inds=(), right_inds=(), virtuali
     s = Tensor(s, [virtualind])
     Vt = Tensor(reshape(conj(V), right_sizes..., size(V, 2)), [right_inds..., virtualind])
 
+    if maxdim !== nothing
+        U=view(u, virtualind => 1:maxdim)
+        s=view(s, virtualind => 1:maxdim)
+        Vt=view(v, virtualind => 1:maxdim)
```

Comment:
just some formatting so that it passes the format check
```suggestion
        U = view(u, virtualind => 1:maxdim)
        s = view(s, virtualind => 1:maxdim)
        Vt = view(v, virtualind => 1:maxdim)
```